src/library/Uno.Toolkit.WinUI.Themes.Markup/Theme.Styles.cs

```diff
@@ -0,0 +1,207 @@
+﻿using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.MarkupHelpers.Internals;
+using Uno.Themes.Markup;
+
+namespace Uno.Toolkit.Themes.Markup
+{
+	public static partial class Theme
+	{
+		public static class Styles
+		{
+			public static class AppBarButton
+			{
+				[ResourceKeyDefinition(typeof(Style), "MainCommandStyle", TargetType = typeof(AppBarButton))]
+				public static ResourceValue<Style> Default => new("MainCommandStyle");
```

Comment:
maybe we could have keep the original name, and then have a default alias that points to it:
```cs
[...] public static Default => Main;
[...] public static Main => new(...);
```